src/GoogleFitOnFhir/Clients/GoogleFit/GoogleFitTelemetryLogger.cs

```diff
@@ -0,0 +1,81 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.ApplicationInsights;
+using Microsoft.Health.Logging.Telemetry;
+using Metric = Microsoft.Health.Common.Telemetry.Metric;
+
+namespace GoogleFitOnFhir.Clients.GoogleFit
+{
+    public class GoogleFitTelemetryLogger : ITelemetryLogger
```

Comment:
I actually think that the TelemetryLogger will not be scoped to just Google Fit. This will be a project wide class. Can the name be just TelemetryLogger?

Reply:
ok